Automattic/jetpack
# [Package] Backup: modify review request logic

#### Changes proposed in this Pull Request:

Currently, we ask for a Jetpack Backup plugin review in the WordPress.org plugin repo whenever:
- the site completes a restore
- the site has a 3+ months active subscription.

Proposals on this PR: 
- Remove the 3+ months active subscription review request 
- Add a new review request whenever a site has 5+ successful backups in a row.  ( latest 5 backups )



#### Jetpack product discussion


P2 discussion: pbNhbs-3Il
Asana task: 1199562518309922-as-1202898039009401


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
- Create a site or use a preexisting site with a Jetpack Backup plan
- Optional but helpful: open the Rewind Debugger for the site to start backups if needed and see the 5 latest backups
- If you don't have 5 successful backups, confirm that the review request is not visible. If you do have them
- Complete 5 successful backups and confirm that the review request is visible
- Click on the review link and confirm that it works
- Dismiss the review request ( click on Maybe later ) and confirm that the review request disappears
- Reload the page and confirm that the message is not displayed anymore

#### Expected component:
<img width="951" alt="Screen Shot 2022-09-01 at 14 10 20" src="https://user-images.githubusercontent.com/16329583/187914269-767b6845-d1fe-4352-931b-6cf578a0d9d2.png">
